reorg of staking technical docs (#142)
onflow/flow: docs/content/token/staking/transactions.mdx

---
title: Staking Transactions Reference

> sidebar_title: Staking Transactions Reference

Reviewer: ```suggestion
```

`sidebar_title` is optional -- it overrides the value of `title`, so isn't needed in this case

---

> # Overview

Reviewer: TODO?

Author: Oops, yes I just forgot to write it. I'll add an overview

|**`SC.06`**| Get Total Flow Staked                  | [idTableStaking/get_total_staked.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/get_total_staked.cdc) |
|**`SC.07`**| Get Total Flow Staked by Node Type     | [idTableStaking/get_total_staked_by_type.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/get_total_staked_by_type.cdc) | 
|**`SC.08`**| Get All Info about a single NodeID     | [idTableStaking/get_node_info.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/get_node_info.cdc) |
|**`SC.09`**| Get a nodes total Commitment           | [idTableStaking/get_node_total_commitment.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/get_node_total_commitment.cdc) |
|**`SC.10`**| Get All Info about a single Delegator  | [idTableStaking/delegation/get_delegator_info.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/idTableStaking/delegation/get_delegator_info.cdc) |

## Node Staker Transactions

> Please see the locked tokens documentation for information for staking with locked tokens.

Reviewer: ```suggestion
Please see the [locked tokens documentation](../locked/transactions) for information about staking with locked tokens.
```

Docs for staking with unlocked tokens will be updated when unlocked tokens are available to stake with.

## Delegator Transactions

> Please see the locked tokens documentation for information for delegatin with locked tokens.

Reviewer: ```suggestion
Please see the [locked tokens documentation](../locked/transactions) for information about delegating with locked tokens.
```
